uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -50,166 +90,173 @@
     <string name="kh_uisdk_adyen_payment_error_40">The payment network can\'t be reached. Retry the transaction with a different payment method.</string>
     <string name="kh_uisdk_adyen_payment_error_41">A PIN or signature is required. Retry the transaction.</string>
     <string name="kh_uisdk_adyen_payment_error_42">The 3D Secure authentication failed due to an issue at the card network or issuer. Retry the transaction, or retry the transaction with a different payment method.</string>
+    <string name="kh_uisdk_airport_bookingMessage">Please enter a valid flight number</string>
     <string name="kh_uisdk_allocating_trip_one">Please wait whilst you\'re allocated a driver</string>
     <string name="kh_uisdk_allocating_trip_two">This may take a moment or two</string>
     <string name="kh_uisdk_allocation_delay_text">It is taking longer than usual for the fleet to allocate a driver. Please wait, contact the fleet, or cancel and rebook with a different fleet.</string>
     <string name="kh_uisdk_allocation_delay_title">Allocation delay</string>
     <string name="kh_uisdk_alternative">ALTERNATIVE</string>
     <string name="kh_uisdk_arrival">Arrival</string>
     <string name="kh_uisdk_arrived">Your %s driver has arrived</string>
+    <string name="kh_uisdk_back">Back</string>
     <string name="kh_uisdk_baggage_max">%d luggages max</string>
+    <string name="kh_uisdk_baggage_max_ios">%1$s luggages maximum</string>
```

Comment:
Is `luggages` grammatically correct? Isn't it `pieces of luggage`? cc: @olivierkarhoo